chainer_chemistry/training/extensions/roc_auc_evaluator.py

```diff
@@ -132,7 +140,19 @@ def evaluate(self):
         # --- set positive labels to 1, negative labels to 0 ---
         pos_indices = numpy.in1d(t_total, self.pos_labels)
         t_total = numpy.where(pos_indices, 1, 0)
-        roc_auc = metrics.roc_auc_score(t_total, y_total)
+        try:
+            roc_auc = metrics.roc_auc_score(t_total, y_total)
+        except ValueError as e:
+            # `ValueError` is usually caused by the following
```

Comment:
It seems you list only one situation where `ValueError` is raised. So, `by the following` seems strange.